shared/chat/conversation/header-area/normal/index.types.js.flow

```diff
@@ -4,7 +4,7 @@ export type Props = {|
   canOpenInfoPanel: boolean,
   channelName: ?string,
   muted: boolean,
-  onCancelPending: () => void,
+  onCancelPending: null | (() => void),
```

Comment:
I'm not exactly sure what onCancelPending is being used for but in all of the conversations I had, it was equal to null. It displays an extra close icon on desktop only if it is defined.

Reply:
It's used when you're in a pending conversation with a fixed set of users. You can get there by clicking a chat button on a profile or team member list.